modules/acid/src/test/ts/browser/TransformationsTest.ts

```diff
@@ -0,0 +1,54 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Obj } from '@ephox/katamari';
+import { assert } from 'chai';
+import * as fc from 'fast-check';
+import * as Transformations from 'ephox/acid/api/colour/Transformations';
+
+describe('TransformationsTest', () => {
+  context('anyToHex', () => {
```

Comment:
Maybe add a test for an arbitrary non-colour string to test it fails gracefully?

Reply:
I could have sworn I added that... oh well it's there now, thanks!